src/App.js

```diff
@@ -28,10 +45,14 @@ const App = () => {
             <Home setToken={setToken} />
           </Route>
           <Route exact path="/list">
-            {token ? <List token={token} /> : <Redirect to="/" />}
+            {token && listItems ? <List token={token} /> : <Redirect to="/" />}
```

Comment:
For this section, I'm curious why `&& listItems` was added. Since it has an initial value of [], this should always come back as `true`. In addition to passing the token, you should be able to pass the listItems down so that you don't have to query again in `List.js`.
```suggestion
            {token ? <List token={token} listItems ={listItems} /> : <Redirect to="/" />}
```

Reply:
Hi @laurenyz I think that was actually an accident, leaving `&& listItems` in. I think we were simply testing out some things and that snuck in! I can get that removed soon.

Reply:
I have removed `&& listItems` from line 48